tests/test_examples.py

```diff
@@ -0,0 +1,46 @@
+import glob
+import os
+import shutil
+
+import pytest
+
+from rapids_dependency_file_generator.rapids_dependency_file_generator import (
+    main as dfg,
+)
+
+CURRENT_DIR = os.path.dirname(__file__)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def clean_actual_files():
+    for root, _, _ in os.walk("tests"):
+        if os.path.basename(root) == "actual":
+            shutil.rmtree(root)
+
+
+def test1():
+    test_dir = os.path.join(CURRENT_DIR, "examples", "test1")
+    expected_dir = os.path.join(test_dir, "output", "expected")
+    actual_dir = os.path.join(test_dir, "output", "actual")
+    dep_file_path = os.path.join(test_dir, "dependencies.yaml")
+
+    dfg(dep_file_path)
+
+    expected_file_set = {
+        os.path.relpath(f, expected_dir)
```

Comment:
Write a function that takes `expected_dir` and returns the expected files. That would be reusable for `expected` and `actual`.